coc/miscmodels.py

```diff
@@ -580,6 +580,40 @@ def __init__(self, *, data, client):
         self.badge = try_enum(Icon, data=data.get("iconUrls"), client=self._client)
 
 
+class CapitalDistrict:
+    """Represents a Clan Capital District.
+
+    Attributes
+    -----------
+    id:
+        :class:`int`: The district's unique ID as given by the API.
+    name:
+        :class:`str`: The district's name.
+    hall_level:
+        :class:`int`: The district's hall level
+    """
+
+    __slots__ = ("id", "name", "_client", "hall_level")
+
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        attrs = [("id", self.id), ("name", self.name)]
+        return "<%s %s>" % (self.__class__.__name__, " ".join("%s=%r" % t for t in attrs),)
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.id == other.id
```

Comment:
Should this compare level too? What would you expect this to do?
```py
if clan1.district_hall[0] == clan2.district_hall[0]:
    print('district halls are the same!')

Reply:
@doluk

Reply:
[d6ad24efe7872616faa355061cfb6dc80198f0d5](https://github.com/mathsman5133/coc.py/commit/d6ad24efe7872616faa355061cfb6dc80198f0d5)